app/api/footprints_api.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class FootprintsApi < Grape::API
+  helpers AuthorizationHelper
+  before do
+    token = headers.fetch('auth_token', nil)
+    authorize_user(token) if token
+  end
+
+  resource :footprints do
+    route_param :id do
+      # GET /footprints/:id
+      desc 'Get footprint' do
+        tags %w[footprints]
+        http_codes [
+          { code: 200, message: 'Footprint' },
+          { code: 404, message: 'Footprint not found' }
+        ]
+      end
+      desc 'Headers', {
+        headers: {
+          'auth_token' => {
+            description: 'Validates your identity',
+            optional: true
+          }
+        }
+      }
+      params do
+        requires :id, type: Integer
+      end
+      get do
+        cars_carbon_footprint = Footprint.find(params[:id]).cars.sum(&:carbon_footprint)
+        public_transports_carbon_footprint = Footprint.find(params[:id]).public_transports.sum(&:carbon_footprint)
+        flights_carbon_footprint = Footprint.find(params[:id]).flights.sum(&:carbon_footprint)
+        house_carbon_footprint = Footprint.find(params[:id]).house.carbon_footprint
+        food_carbon_footprint = (Footprint.find(params[:id]).food.min_carbon_footprint +
+                                 Footprint.find(params[:id]).food.max_carbon_footprint) / 2
```

Comment:
@UrsuAnastasia we might want to display both the minimum and the maximum carbon footprint as they are computed based on the user's answers.